### PR TITLE
feat(component): implemented helper method for MEXICAN SPANISH

### PR DESCRIPTION
**Pull request by AvgoustisW**
implemented a helper method for MEXICAN SPANISH fix issue [#35](https://github.com/chgasparoto/aws-polly-voices/issues/35)

Add a helper method for Mexican Spanish

## Proposed changes

Hello, I have implemented Mexican Spanish helper to get into the GitHub flow and Hacktoberfest, have a good one and thanks for doing this!

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/chgasparoto/aws-polly-voices/blob/master/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

mexican_spanish

- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by AvgoustisW** [comment]
Hello chgasparoto,

I suppose this has been finished : )

Let me know and thanks!

File: README.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ const portuguese = voices.portuguese()
 const dutch = voices.dutch()
 const russian = voices.russian()
 const german = voices.german()
+const mexican = voices.mexican()
```
Comment (chgasparoto) [inline README.md]: this should be in the `language and country` block because it's related to a specific country :)

Comment (AvgoustisW) [reply README.md]: Done

**Comment by chgasparoto** [comment]
@AvgoustisW I just merged other PR into master. Could you please fix the conflict?

Please, in order to fix the issue automatically put **fix #issue_number**

**Comment by AvgoustisW** [comment]
> @AvgoustisW I just merged other PR into master. Could you please fix the conflict?
> 
> Please, in order to fix the issue automatically put **fix #issue_number**

Conflict has been fixed. I am not sure I entirely understand what you mean by putting the fix #issue number. I have it on my PR description.

Let me know, and thanks!

**Comment by chgasparoto** [comment]
@AvgoustisW what I meant: **fix issue #issue_number** (like in your PR description) doesn't work. You must put it like this: **fix #issue_number**. Now I have to close the issue manually :)

do you get it?

**Pull request merged**